kartograf/coverage.py

```diff
@@ -11,7 +11,11 @@ def coverage(map_file, ip_list_file):
     rpki_masks = []
     for line in map_file:
         pfx, _ = line.split()
-        ipn = ipaddress.ip_network(pfx)
+        try:
+            ipn = ipaddress.ip_network(pfx)
+        except ValueError:
+            print(f"Invalid IP network provided: {line}")
+            continue
```

Comment:
I think it would make sense to stop here as well just for the other file

Reply:
I updated this to raise, but assuming this is from an ASmap file generated by kartograf, it's unlikely to occur since we already ensure validity of networks we write.

Reply:
> assuming this is from an ASmap file generated by kartograf, it's unlikely to occur since we already ensure validity of networks we write.

Thanks! Right, but if something is wrong here that's more reason to me to fail loudly and make the user aware that they likely have made a mistake rather than showing them a result that isn't of use to them.